pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>logstash-logback-encoder</artifactId>
             <version>6.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.sql2o</groupId>
+            <artifactId>sql2o</artifactId>
+            <version>1.5.4</version>

```

Comment:
can we use a newer version of this library?

Reply:
I checked the version on https://mvnrepository.com/  , changing it to 1.6.0